src/pages/index.js

```diff
@@ -12,14 +12,30 @@ import {
   profileEditButton,
   profileAvatarButton,
   profileTitleInput,
-  profileAvatar,
   profileDescriptionInput,
 } from "../utils/constants.js";
 import Api from "../components/Api.js";
 /////////////////////////////////////////////////////////////////////\
 
 // =========  FORM VALIDATORS  ========= //
 
+// stores form validators
+const formValidators = {};
+
+const enableValidation = (config) => {
+  const formList = Array.from(document.querySelectorAll(config.formSelector));
+  formList.forEach((formElement) => {
+    const validator = new FormValidator(config, formElement);
+    // here you get the name of the form
+    const formName = formElement.getAttribute("name");
+
+    // here you store the validator using the `name` of the form
+    formValidators[formName] = validator;
+    validator.enableValidation();
+  });
+};
+enableValidation(settings); // calls enable validation
```

Comment:
Please, delete `enableValidation` because you create 3 separate form validators

Reply:
ah, of course. thank you.